### ISSUE TITLE
Consistency with respect to wording "extraction" of component key

### ISSUE THREAD

**Issue by falko-strenzke** [openpgp-pqc/draft-openpgp-pqc#15]
# Falko's original remark

In Section 4.1.2 we state

> Extract the component public key kyberPublicKey that is part of the recipient's composite public key

This seems inconsistent in wording with the descriptions for the EC-KEMs, where no mentioning of the extraction of the component public key from the composite key is made. IMO it should be mentioned in for each scheme or for none.

# Aron's comment on this (from previous issue)

> In Section 4.1.2
>This seems inconsistent in wording with the descriptions for the EC-KEMs, 

In the ECC sections we mention 
```...where R is the component public key eccPublicKey...```
I would consider removing point (1) from 4.1.2 and rewrite
``` 
2. Invoke (kyberCipherText, keyShare) <- kyberKem.encap(kyberPublicKey) 
   where kyberPublicKey is extracted from recipient's composite public key
```

**Comment by falko-strenzke** [comment]
Meeting 2023-01-19:

* Aron & Falko: just about consistency, will be aligned

**Comment by falko-strenzke** [comment]
Meeting 2023-06-05:
* AW will work on this

**Comment by falko-strenzke** [comment]
will be fixed by #80

**Comment by falko-strenzke** [comment]
fixed via #80
